src/api/package.js

```diff
@@ -50,6 +50,12 @@ export default async (providedOptions = {}) => {
 
   const packageJSON = await readPackageJSON(dir);
 
+  if (path.dirname(require.resolve(path.resolve(dir, packageJSON.main))) === dir) {
+    console.error(`${packageJSON.main}`.red);
```

Comment:
Perhaps:

```javascript
console.error(`Entry point: ${packageJSON.main}`.red);
```

Just kind of weird to have a file name without context in stderr, although I realize the error thrown is contextual in the next line.